cmd/buildah/containers.go

```diff
@@ -261,12 +261,24 @@ func containerOutputUsingTemplate(format string, params containerOutputParams) e
 
 func containerOutputUsingFormatString(truncate bool, params containerOutputParams) {
 	if truncate {
-		fmt.Printf("%-12.12s  %-8s %-12.12s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, params.ImageName, params.ContainerName)
+		fmt.Printf("%-12.12s  %-8s %-12.12s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, truncateString(params.ImageName, 32), params.ContainerName)
 	} else {
-		fmt.Printf("%-64s %-8s %-64s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, params.ImageName, params.ContainerName)
+		fmt.Printf("%-64s %-8s %-64s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, truncateString(params.ImageName, 32), params.ContainerName)
```

Comment:
Truncating part of the output when `truncate` is false (i.e., the user specified `--notruncate`) seems like a bug.

Reply:
I see, I changed that part back.